File: C-5-SDT/OOP_and_Python_Programming/M3.5/K_Sum_Digits.py
# https://codeforces.com/group/MWSDmqGsZm/contest/219774/problem/K
# solution 1
# n = int(input())
# digs = input()
# dig_lst = []
# for i in range(n):
#     dig_lst.append(int(digs[i]))
# sum_digs = sum(dig_lst)
# print(sum_digs)

# solution 2
n = int(input())
digs = input()
sum=0
for i in range(n):
    # sum += int(digs[i])
    sum += ord(digs[i]) - ord('0')
print(sum)

File: C-1-C/M11.5/K_Sum_Digits.c
// https://codeforces.com/group/MWSDmqGsZm/contest/219774/problem/K
#include <stdio.h>
int main()
{
    int n, sum = 0;
    scanf("%d", &n);

    // char s[1000001]; // 1e6 for characters + 1 for null terminator
    char s[n + 1];  // Dynamic size based on input n
    scanf("%s", s); // &s is not needed for char array

    for (int i = 0; i < n; i++)
    {
        sum += s[i] - '0'; // Convert char to int by subtracting '0'
    }
    printf("%d\n", sum);
    return 0;
}